packages/kbn-config-schema/src/index.ts

```diff
@@ -65,6 +68,14 @@ function boolean(options?: TypeOptions<boolean>): Type<boolean> {
   return new BooleanType(options);
 }
 
+function buffer(options?: TypeOptions<Buffer>): Type<Buffer> {
```

Comment:
__note:__ please don't forget to document these new methods once https://github.com/elastic/kibana/pull/50307/ is merged.